media-fonts/iosevka/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>david.truby@gmail.com</email>
+		<name>David Truby</name>
+	</maintainer>
+	<upstream>
```

Comment:
Please add proxy-maint team below you as well (you can find an example how in other packages or wiki).